### PR TITLE
Dir4loc

### PR DESCRIPTION
**Pull request by thorunna**

### CODE REVIEW

**Comment by vthorsteinsson** [comment]
I resolved a final conflict @thorunna - are you OK with merging this?

**Comment by thorunna** [comment]
Yes, thank you!

**Comment by vthorsteinsson** [comment]
OK, there are a couple of mypy type checking errors - I'll fix those before merging...

**Comment by vthorsteinsson** [comment]
@thorunna  Can you look at the lines I commented with `FIXME` in pattern.py?

**Comment by thorunna** [comment]
Will do!

File: src/reynir_correct/pattern.py [hunk 1]
```diff
@@ -917,6 +920,226 @@ def vera_að(self, match: SimpleTree) -> None:
...
+            else:
+                narrow_match = adv.tidy_text + ' ' + pp.tidy_text
+        elif pp.span < adv.span:
+            narrow_match = pp.tidy_text + ' ' + adv.tidy_text
+        correction = adv.tidy_text+'i'
+        text = "Hér á líklega að vera {0} í stað {1}".format(
+            correction, adv.tidy_text
+        )
+        detail = "Í samhenginu '{0}' er rétt að nota atviksorðið '{1}' í stað '{2}'.".format(
+            narrow_match, correction, adv.tidy_text
```
Comment (vthorsteinsson) [inline src/reynir_correct/pattern.py:943]: Pylance complains that `narrow_match `is possibly unbound - it is not guaranteed to have received a value from the if statements above.

**Comment by vthorsteinsson** [comment]
@thorunna  Can you take a look at the lines commented with `FIXME` and `TODO` in pattern.py and test_patterns.py - there are some things there left to clarify/modify?

**Comment by vthorsteinsson** [comment]
OK to merge @thorunna ?

**Comment by thorunna** [comment]
Yes!

**Pull request merged**